src/Selenium2Library/keywords/cookie.py

```diff
@@ -1,46 +1,58 @@
-from .keywordgroup import KeywordGroup
+from Selenium2Library.robotlibcore import keyword
 
 
-class CookieKeywords(KeywordGroup):
+class CookieKeywords(object):
 
+    def __init__(self, ctx):
+        self.ctx = ctx
+
+    @keyword
     def delete_all_cookies(self):
         """Deletes all cookies."""
-        self._current_browser().delete_all_cookies()
+        self.ctx.current_browser().delete_all_cookies()
```

Comment:
Good example where `ctx.browser.delete_all_cookies()` would be cleaner than the current `ctx.current_browser().delete_all_cookies()`.